components/infobox/commons/infobox_team.lua

```diff
@@ -169,13 +169,17 @@ function Team:_createLocation(location)
 			'[[:Category:' .. location .. '|' .. location .. ']]'
 end
 
+function Team:_getStandardLocationValue(location)
```

Comment:
Halo currently use _getStandardLocationValue() in their custom outside the saving of location/location2; in their region handling
Should they keep have a CustomTeam:_getStandardLocationValue(), should we change Team:_getStandardLocationValue() to public instead of private? Or a third option?

Reply:
i think making it public is okay